cypress/integration/login.js

```diff
@@ -1,27 +1,29 @@
-describe('Sourcebooks login', function() {
+import LoginPage from '../objects/loginPage'
 
-    it('Should display validation for empty user after attempted loggin', function () {
-        
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')
-    })
+describe('Sourcebooks login', function() {

```

Comment:
Apparently you've replaced the login test with a task creation test, but left the old name both in the test description and in the file name.